style.css

```
@@ -6,7 +6,7 @@ Author URI: https://wordpress.org
 Description:
 Requires at least: 5.9
 Tested up to: 6.1
-Requires PHP: 5.7
+Requires PHP: 7.4
```

WordPress 6.1 has the minimium PHP version 5.6, so you should be able to install the theme with this version.

```suggestion
Requires PHP: 5.6
```